src/nrnpython/setup.py.in

```diff
@@ -122,7 +122,10 @@ libdirs = [destdir + get_escaped_path("@NRN_LIBDIR@"),
 
 # prepare rpath flags for neuron and iv libs directories
 rpath_prefix_flag='-Wl,-R'
-extra_rpath_flags = [rpath_prefix_flag+lib_path for lib_path in libdirs]
+extra_link_args = [@NRN_LINK_FLAGS_COMMA_SEPARATED_STRINGS@]
```

Comment:
(bit more questions on this file as this has been fragile in non-trivial setup)


@olupton : to clarify - the idea is to bring additional linker flag that is needed for sanitizer link line?